<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>W_DistanceListClass</class>
 <widget class="QMainWindow" name="W_DistanceListClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>W_DistanceList</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="startingStadiumLabel">
        <property name="font">
         <font>
          <family>Yu Gothic Medium</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Starting Stadium</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="endingStadiumLabel">
        <property name="font">
         <font>
          <family>Yu Gothic Medium</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ending Stadium</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="distanceLabel">
        <property name="font">
         <font>
          <family>Yu Gothic Medium</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Distance</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
